cloud/vsphere/cmd/vsphere-machine-controller/main.go

```diff
@@ -1,12 +1,9 @@
 /*
 Copyright 2018 The Kubernetes Authors.
-
```

Comment:
The new lines are needed so the boilerplate corresponds to https://github.com/kubernetes-sigs/cluster-api/blob/master/boilerplate.go.txt

Reply:
oops, I'll fix this. Thanks!